src/coord/config.c

```diff
@@ -149,6 +176,19 @@ CONFIG_SETTER(setOSSACLUsername) {
   RETURN_STATUS(acrc);
 }
 
+// acl-username
+RedisModuleString * get_oss_acl_username(const char *name, void *privdata) {
+  char *str = *(char **)privdata;
+  if (str) {
```

Comment:
Can null check be dropped? (if initialized properly)

Reply:
Yes. Check dropped.